model/k8s/metrics.yaml

```diff
@@ -63,6 +63,28 @@ groups:
       - ref: network.interface.name
       - ref: network.io.direction
 
+  # k8s.container.* metrics
+  - id: metric.k8s.container.status.state
+    type: metric
+    metric_name: k8s.container.status.state
+    stability: experimental
+    brief: ""
+    instrument: updowncounter
+    unit: "{container}"
+    attributes:
+      - ref: k8s.container.status.state
+        requirement_level: required
+  - id: metric.k8s.container.status.reason
+    type: metric
+    metric_name: k8s.container.status.reason
+    stability: experimental
+    brief: ""
```

Comment:
Same here a `brief` should be provided unless there is a reason for not doing so.